packages/knip/src/ConfigurationValidator.ts

```diff
@@ -77,6 +77,7 @@ export const pluginSchema = z.union([
 ]);
 
 const pluginsSchema = z.object({
+  xo: pluginSchema,
```

Comment:
Pretty sure this object should be in alphabetical order?

Reply:
it's generated code, and biome didn't complain about it, but you're absolutely right, it doesn't look right there.